src/tiledimage.js

```diff
@@ -277,14 +280,30 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
         });
     },
 
+    /**
+     * Forces the system consider all tiles in this tiled image
+     * as outdated, and fire tile update event on relevant tiles
+     * Detailed description is available within the 'tile-invalidated'
+     * event.
+     * @param {Boolean} [restoreTiles=true] if true, tile processing starts from the tile original data
+     * @param {boolean} [viewportOnly=false] optionally invalidate only viewport-visible tiles if true
+     * @param {number} [tStamp=OpenSeadragon.now()] optionally provide tStamp of the update event
+     */
+    requestInvalidate: function (restoreTiles = true, viewportOnly = false, tStamp = $.now()) {
+        const tiles = viewportOnly ? this._lastDrawn.map(x => x.tile) : this._tileCache.getLoadedTilesFor(this);
+        return this.viewer.world.requestTileInvalidateEvent(tiles, tStamp, restoreTiles);
+    },
+
     /**
      * Clears all tiles and triggers an update on the next call to
      * {@link OpenSeadragon.TiledImage#update}.
      */
     reset: function() {
         this._tileCache.clearTilesFor(this);
+        this._currentMaxTilesPerFrame = this.maxTilesPerFrame * 10;
```

Comment:
The problem is that `this._tileCache.clearTilesFor(this);` makes sure there is no data. So immediatelly re-draw would draw nothing.

Reply:
I guess I don't literally mean *immediately* but, rather, as soon as there is data in the cache for a tile it would be redrawn without having to go through the tile loading order. This would happen if the `reset` was called in conjunction with changing a filter for example. When the async processing is done on the cached tiles they would automatically be redrawn on the next frame. Then, requests to load additional tiles could/should proceed as normal.

Reply:
I don't understand. 

Returning to the original queststion, yes, this is to make the data load faster. Even so when the data might not be downloaded at all, but fetched from zombies. If you don't set this, the you get one tile, redraw, get another, redraw. In fact, it might be even: get one tile, redraw, proces tile, redraw, convert tile, redraw, get second tile.... It implements the `_currentMaxTilesPerFrame ` which slowly declines back to the original value, this is just temporal boost.